.github/actions/setup/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Setup
```

Comment:
Sounds fine to me to add this github actions and templates, same one as `create-react-native-library`, but if you don't want them tell me, and I'll remove this changes